modular_citadel/code/modules/mob/living/silicon/robot/dogborg_equipment.dm

```diff
@@ -167,7 +167,7 @@ SLEEPER CODE IS IN game/objects/items/devices/dogborg_sleeper.dm !
 	if(!proximity)
 		return
 	do_attack_animation(target, null, src)
-	user.visible_message("<span class='notice'>[user] [pick(attack_verb)] \the [target.name] with their nose!</span>")
+	user.visible_message("<span class='notice'>[user] [attack_verb] \the [target.name] with their nose!</span>")
```

Comment:
this is going to make it display a list whenever you hit something with the nose...

Reply:
it should pick one of them right?